taxi/views.py

```diff
@@ -1,8 +1,13 @@
+from http.client import HTTPResponse
 from django.contrib.auth.decorators import login_required
+from django.http import HttpRequest, HttpResponseRedirect
```

Comment:
The import `HttpRequest` from `django.http` is not used in this file. Consider removing it to clean up the code.